inspirehep/modules/authors/forms.py

```diff
@@ -563,6 +563,12 @@ def __init__(self, *args, **kwargs):
         """Constructor."""
         super(AuthorUpdateForm, self).__init__(*args, **kwargs)
         is_review = kwargs.pop('is_review', False)
+        is_update = kwargs.pop('is_update', False)
```

Comment:
Woah, this trusts whatever is in the user request to tell us what to validate? https://github.com/inspirehep/inspire-next/blob/4d1f980894252e38459c1a9dc15762cc169cb504/inspirehep/modules/authors/views.py#L189